build.gradle

```diff
@@ -370,7 +370,7 @@ configurations.all {
 
 dependencies {
 
-  implementation 'com.github.hmcts:civil-commons:1.0.36'
+  implementation 'com.github.hmcts:civil-commons:1.0.39'
```

Comment:
Civil-commons changes didn't go through the version 1.0.38 therefore increasing the version with my changes